tsoa.json

```diff
@@ -4,9 +4,17 @@
   "controllerPathGlobs": ["src/controllers/**/*.ts"],
   "spec": {
     "outputDirectory": "src",
-    "specVersion": 3
+    "specVersion": 3,
+    "securityDefinitions": {
+      "bearerAuth": {
+        "type": "http",
+        "scheme": "bearer",
+        "bearerFormat": "JWT"
+      }
+    }
   },
   "routes": {
-    "routesDir": "src"
+    "routesDir": "src",
+    "authenticationModule": "./src/authentication.ts"
```

Comment:
Mmm... since it's a module maybe lets put it in lib/? since it's around 10 lines of code maybe include into `server.ts` and then when we have an x number of middleware we can break down. either way... if possible without hustle would prefer it to not be at root of `src/` but again this is not stopping me from approving.

Reply:
moved to `/lib`